ajax/libs/background-check/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "background-check",
+  "filename": "background-check.min.js",
+  "description": "Automatically switch to a darker or a lighter version of an element depending on the brightness of images behind it.",
+  "homepage": "http://www.kennethcachia.com/background-check/",
+  "keywords": [
+    "background",
+    "check"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kennethcachia/background-check.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://gitlab.com/kennethcachia/background-check.git",
```

Comment:
I think this should be `git://github.com/kennethcachia/background-check.git`
not GitLab